CodeStarterCamp_Week2/main.swift

```diff
@@ -8,12 +8,20 @@
 
 import Foundation
 
-// practice
+// MARK: - practice
 //choose(gender: Gender.man)
 //choose(cardinalPoints: .west)
+//optionalPractice1()
+//optionalPractice2()
+//optionalPractice3()
 
-// step 2
-let myLottoNumbers: [Int] = [1, 2, 3, 4, 5, 6]
-compareLotteryNumbers(target: myLottoNumbers, to: generateLotteryNumbers())
-
+// MARK: - step2
+//let myLottoNumbers: [Int] = [1, 2, 3, 4, 5, 6]
+//compareLotteryNumbers(target: myLottoNumbers, to: generateLotteryNumbers())
 
+// MARK: - step3
+var lotteryHistory: [String: Array<Int>] = [:]
```

Comment:
요건 사람마다 다르긴 합니다만, 저는 요렇게 표현하는 걸 더 선호합니다.
이 코드가 틀렸다는 것은 아니고, 요런 방법도 있구나~ 정도로 알고 넘어가시면 되요~
```suggestion
var lotteryHistory: [String: [Int]] = [:]
```